src/code/z_elf_message.c

```diff
@@ -1,46 +1,60 @@
 #include "global.h"
+#include "z64elf_message.h"
 
-// TODO: use macros to define elf messages once the format is fully documented
 ElfMessage sChildSariaMsgs[] = {
-    { 0x66, 0x00, 0x03, 0x00 }, { 0x00, 0x37, 0x61, 0x00 }, { 0xE0, 0x00, 0x64, 0x00 }, { 0x00, 0x25, 0x62, 0x00 },
-    { 0x00, 0x37, 0x63, 0x00 }, { 0x00, 0x43, 0x65, 0x00 }, { 0x06, 0x30, 0x66, 0x66 }, { 0x06, 0x30, 0x66, 0x67 },
-    { 0x06, 0x30, 0x66, 0x68 }, { 0x06, 0x20, 0x67, 0x65 }, { 0x06, 0x30, 0x68, 0x69 }, { 0x06, 0x30, 0x68, 0x6A },
-    { 0xE0, 0x00, 0x69, 0x00 },
+/* 0000   0 */ ELF_MESSAGE_STRENGTH_UPG(BRANCHING, 3, false, 0), // TODO goron bracelet
+/* 0004   1 */ ELF_MESSAGE_FLAG(CHECK, 0x0161, false, 0x37), /* eventChkInf[3] & 0x80 */
+/* 0008   2 */ ELF_MESSAGE_END(0x0164),
+
+/* 000C   3 */ ELF_MESSAGE_FLAG(CHECK, 0x0162, false, 0x25), /* eventChkInf[2] & 0x20 */
+/* 0010   4 */ ELF_MESSAGE_FLAG(CHECK, 0x0163, false, 0x37), /* eventChkInf[3] & 0x80 */
+/* 0014   5 */ ELF_MESSAGE_FLAG(CHECK, 0x0165, false, 0x43), /* eventChkInf[4] & 0x8 */
+/* 0018   6 */ ELF_MESSAGE_MEDALLION(CHECK, 0x0166, false, ITEM_MEDALLION_FOREST),
+/* 001C   7 */ ELF_MESSAGE_MEDALLION(CHECK, 0x0166, false, ITEM_MEDALLION_FIRE),
+/* 0020   8 */ ELF_MESSAGE_MEDALLION(CHECK, 0x0166, false, ITEM_MEDALLION_WATER),
+/* 0024   9 */ ELF_MESSAGE_SONG(CHECK, 0x0167, false, ITEM_SONG_STORMS),
+/* 0028  10 */ ELF_MESSAGE_MEDALLION(CHECK, 0x0168, false, ITEM_MEDALLION_SPIRIT),
+/* 002C  11 */ ELF_MESSAGE_MEDALLION(CHECK, 0x0168, false, ITEM_MEDALLION_SHADOW),
+/* 0030  12 */ ELF_MESSAGE_END(0x0169),
```

Comment:
I feel like the comments for each entry are unnecessary and could be omitted to keep things clean and easy to work with in modding purposes